### ISSUE TITLE
Use of publicDnsName should failover to IP if no name is available

### ISSUE THREAD

**Issue by mes5k** [nextflow-io/nextflow#240]
If nextflow cloud is run in a private VPC then no public DNS name will be associated with the master server, which means that the output printed by [CmdCloud](https://github.com/nextflow-io/nextflow/blob/master/src/main/groovy/nextflow/cli/CmdCloud.groovy#L248) will look like this with the host name blank:

```
...
Login in the master node using the following command: 
  ssh -i <path to msmoot-aws key file> ec2-user@
```

It would be better to failover to an IP address.

**Comment by pditommaso** [comment]
Do you have a public IP address for that ? or do you access it through the private IP/name ?

**Comment by mes5k** [comment]
No public IP either - I'm accessing it using a private IP.  It looks like a private name is also assigned, so that could be used as well.

**Comment by pditommaso** [comment]
Included in version `0.22.5`
